iam/api/QuickStartV2/QuickStart.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
This should be netcoreapp2.1 which will have long term support from Microsoft.

Reply:
Done